### PR TITLE
Add PPP

### PR DESCRIPTION
**Pull request by angnobel**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ All fields are not subjected to the restriction in the input table and can take
 ### Feature: Candidate Management
 
 Manage a list of candidates for your company, with the simple instructions below!
-Each candidate is uniquely identified their email. 
+Each candidate is uniquely identified their email.
```
Comment (wanyu-l) [inline docs/UserGuide.md:222]: Perhaps it should be "...identified by their email"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ All fields are not subjected to the restriction in the input table and can take
 ### Feature: Candidate Management
 
 Manage a list of candidates for your company, with the simple instructions below!
-Each candidate is uniquely identified their email. 
+Each candidate is uniquely identified by their email.
 Different candidates can have the same name as along as the name, email and phone are not all the same.
```
Comment (JeffZincatz) [inline docs/UserGuide.md:223]: There is a conflict here. Master branch implementation uses only email as the unique identifier.

**Review by JeffZincatz — COMMENTED** [review]
LGTM, just need to resolve the conflicts.

**Review by seaweediman — APPROVED** [review]
LGTM

**Pull request merged**